data/kushvahasumit.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name" : "sumit",
+  "displayStatusPublic" : true,
+  "type" : "personal",
+  "bio" : "Undergraduate || new for open source || enthusiast about tech world || web dev",
+  "avatar" : "https://github.com/kushvahasumit.png",
+  "links" : [
+  {
+  "name" : "follow me on Linkedin",
+  "url" : "https://www.linkedin.com/in/sumit-kumar-4a1344254",
```

Comment:
Appears that the LinkedIn link you have here goes to a 404 page.

Reply:
sorry for all issues, i fixed the linkedin link but i didn't have any idea how to change my code please guide me.

Reply:
Don't say sorry for learning :+1: 
- Go back to your fork (Sync your fork with this upstream repo as that's always good)
- Go the required `json` file and you can directly edit in your Github UI.
- Make all the necessary corrections, and at the bottom you'll be required to to commit those new changes.
- This PR will get updated with your corrections.

If there is any issue, feel free to ask.

Reply:
Something else to note here,

If you've made the changes on your local computer, and not in the Github UI, all you have to do is commit and push. The PR will still automatically get updated.

Reply:
I delete this , and i make a new pr.